### ISSUE TITLE
Results?

### ISSUE THREAD

**Issue by PiotrDabkowski** [sh-lee-prml/BigVGAN#9]
First of all, thank you for the project, I think it is really useful, especially that the official NVIDIA implementation is not released yet!

Did you manage to train the model to satisfactory quality, and replicate the results from the paper?

Would it be possible to include some samples or pretrained model to see how the implementation works in practice?

Thank you!

**Comment by sh-lee-prml** [comment]
Thank you for your interest. 

Actually, due to the computational resource constraints, I stopped training BigVGAN vocoder 😢. 
(I trained it only for 300k steps)

When I evaluated, BigVGAN has much better performance than HiFi-GAN in Mel loss and PESQ metrics (Both in wide and narrow bands) as described in the paper. 

However, in my subjective opinion, the audio quality of BigVGAN is slightly improved than that of HiFi-GAN. 
But, I just evaluated it with seen speaker data and I think that the performance for out-of-distribution samples is much better...!

Now, I will train this model with an end-to-end TTS system... I think it will take 25 days on 4x A100... 

Before that, I would be appreciated if you train it and share the results 👍

**Comment by skol101** [comment]
@sh-lee-prml hello, could you share the results of your E2E training?

**Comment by suvaansh** [comment]
Can you please share the checkpoint for the 300k step training?
